target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-whw01v1.dts

```diff
@@ -0,0 +1,238 @@
+		};
+	};
+
+	reset_pinmux {
+		linux,phandle = <0x52>;
+		phandle = <0x52>;
+
+		mux {
+			pins = "gpio63";
+			bias-pull-up;
+		};
+	};
+};
+
+&spi_0_pins {
+	pinmux_cs {
+		pins = "gpio54", "gpio4";
+	};
+	pinconf_cs {
+		pins = "gpio54", "gpio4";
+	};
+};
+
+&blsp1_spi1 {
+	status = "okay";
+	pinctrl-0 = <&spi_0_pins>;
+	pinctrl-names = "default";
+	cs-gpios = <&tlmm 54 GPIO_ACTIVE_HIGH>, <&tlmm 4 GPIO_ACTIVE_HIGH>;
+
+	nor@0 {
+		spi-max-frequency = <0x16e3600>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <0x01>;
+			#size-cells = <0x01>;
+
+			partition@0 {
+				label = "0:SBL1";
+				reg = <0x00 0x40000>;
+				read-only;
+			};
+
+			partition@40000 {
+				label = "0:MIBIB";
+				reg = <0x40000 0x20000>;
+				read-only;
+			};
+
+			partition@60000 {
+				label = "0:QSEE";
+				reg = <0x60000 0x60000>;
+				read-only;
+			};
+
+			partition@c0000 {
+				label = "0:CDT";
+				reg = <0xc0000 0x10000>;
+				read-only;
+			};
+
+			partition@e0000 {
+				label = "APPSBL";
+				reg = <0xd0000 0xa0000>;
+				read-only;
+			};
+
+			partition@160000 {
+				label = "0:ART";
+				reg = <0x170000 0x10000>;
+				read-only;
+			};
+
+			partition@170000 {
+				label = "u_env";
+				reg = <0x180000 0x40000>;
+			};
+
+			partition@190000 {
+				label = "s_env";
+				reg = <0x1c0000 0x20000>;
+			};
+
+			partition@1b0000 {
+				label = "devinfo";
+				reg = <0x1e0000 0x20000>;
+			};
+		};
+	};
+
+	nand@1 {
+		reg = <1>;
+		status = "ok";
+		compatible = "spi-nand";
+		spi-max-frequency = <0x16e3600>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <0x01>;
+			#size-cells = <0x01>;
+
+			partition@0 {
+				label = "kernel";
+				reg = <0x00 0x5000000>;
+				read-only;
+			};
+
+			partition@600000 {
+				label = "rootfs";
+				reg = <0x600000 0x4a00000>;
+				read-only;
+			};
+
+			partition@5000000 {
+				label = "alt_kernel";
+				reg = <0x5000000 0x5000000>;
+				read-only;
+			};
+
+			partition@5600000 {
+				label = "alt_rootfs";
+				reg = <0x5600000 0x4a00000>;
+				read-only;
+			};
+
+			partition@a000000 {
+				label = "sysdiag";
+				reg = <0xa000000 0x200000>;
+				read-only;
+			};
+
+			partition@a200000 {
+				label = "syscfg";
+				reg = <0xa200000 0x5e00000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&blsp1_i2c3 {
+	status = "okay";
+	pinctrl-0 = <&i2c_0_pins>;
+	pinctrl-1 = <&i2c_0_pins>;
+	pinctrl-names = "i2c_active", "i2c_sleep";
+
+	leds {
+		compatible = "nxp,pca9633";
+		#address-cells = <0x01>;
+		#size-cells = <0x00>;
+		reg = <0x62>;
+
+		system_red {
+			label = "red:system";
+			reg = <0x00>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		system_green {
+			label = "green:system";
+			reg = <0x01>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		led_system_blue: system_blue {
+			label = "blue:system";
+			reg = <0x02>;
+			linux,default-trigger = "default-on";
+		};
+
+		unused {
+			label = "unused";
```

Comment:
no color?

Reply:
I've added this in.

I'm not certain whether it is required as the device has a single RGB LED which is connected to the i2c controller; I'll need to read more into the use of the LED common bindings to better understand how the `color` property is used :)